tests/integration/targets/ntnx_lcm_v2/tasks/lcm_operations.yml

```diff
@@ -0,0 +1,374 @@
+---
```

Comment:
add sample var file content in comments what this test expects from prepare_env/vars.yml